parser.y

```diff
@@ -6436,20 +6437,27 @@ SetExpr:
 EqOrAssignmentEq:
     eq | assignmentEq
 
+VariableName:
+	Identifier
+|	Identifier '.' VariableName
```

Comment:
Should it be `Identifier '.' Identifier`?
```
mysql> set xx.xx.xx=666;
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '.xx=666' at line 1
```

Reply:
Done